Create a shared Wizard component #114

ravighevariya10 opened the pull request:
Created the wizard shared component

ravighevariya10 committed:
Created a wizard shared component

ravighevariya10 committed:
removed the unnecessary imports

ravighevariya10 committed:
removed the extra box

ravighevariya10 committed:
updated the result card color

ravighevariya10 committed:
Updated the styling

ravighevariya10 committed:
changed the color of the result card

ravighevariya10 committed:
changed the color

ravighevariya10 committed:
changed the bg color

ravighevariya10 committed:
changed the styling

ravighevariya10 committed:
Testing

ravighevariya10 committed:
Updated the styling

ravighevariya10 committed:
updated the bg color

ravighevariya10 committed:
updated the styling

ravighevariya10 committed:
updated the styling

ravighevariya10 committed:
updated styling

ravighevariya10 committed:
Updated the styling

ravighevariya10 committed:
Testing

ravighevariya10 committed:
Updated the styling

ravighevariya10 committed:
updated the styling

ravighevariya10 committed:
Final styling changes

YazidZidane commented on src/Wizard/wizard.jsx:
```diff
@@ -0,0 +1,409 @@
+    const [multipleResultFlag, setMultipleResultFlag] = useState(false);
+
+    const appDetails = {
+        vegspec: { text: "Vegspec App", url: "https://vegspec.org", reason: "if you want solutions based on NRCS conservation practices" },
```
The `if` might also need to be capitalized as others.

YazidZidane commented on src/Wizard/wizard.jsx:
```diff
@@ -0,0 +1,409 @@
+                            onClick={handleStart}
+                            buttonType=""
+                            sx={{
+                                background: "#598444",
```
Can you try to use the theme color for most of the colors?

YazidZidane reviewed (commented):
I think there should be a `Next` button next to the `Back` button which would allow to go to the next step if there's a selection existed in current step.

YazidZidane commented on src/Wizard/wizard.jsx:
```diff
@@ -0,0 +1,409 @@
+} from '@mui/icons-material';
+import { styles } from './styles/landingStyles';
+import PSAButton from '../Button'
+import PSAFigmaButton from '../FigmaButton';
```
Can you only use one kind of the shared buttons?

ravighevariya10 committed:
1. Moved the custome styles of Yes, No, and Back buttons
2. Used the one same shared button instead of two
3. Capitalized 'if' in the reason of the Vegspec
4. Merged the PSATheme with the Wizard Application theme

ravighevariya10 committed:
1. Added the new button type 'Next' in the shared PSAButton
2. Added the 'Next' button in the shared wizard

ravighevariya10 committed:
made the styling changes for Back button

ravighevariya10 committed:
changed styling for 'Next' button

YazidZidane commented:
<img width="833" height="374" alt="image" src="https://github.com/user-attachments/assets/e0c00f08-1c49-4f00-8313-6aa7684eb0c5" />
<img width="833" height="374" alt="image" src="https://github.com/user-attachments/assets/ac4e5da1-ec73-4021-97c7-5b5fad6e7f44" />
The Next & Back buttons works well. Can you also make them showing on the same location under each question? Currently the location changes when going through questions.

ravighevariya10 committed:
1. Fixed the position of nav buttons
2. Fixed issue with deselecting the selected answer

ravighevariya10 committed:
made styling changes

ravighevariya10 committed:
width of the card

ravighevariya10 committed:
width of the stacks

ravighevariya10 committed:
made styling changes

ravighevariya10 committed:
made styling changes

ravighevariya10 committed:
made styling changes

ravighevariya10 committed:
made styling changes
